packages/networks/src/networks/index.ts

```diff
@@ -1,16 +1,18 @@
-export * from './localhost'
-export * from './mainnet'
-export * from './gnosis'
-export * from './polygon'
-export * from './bsc'
-export * from './optimism'
-export * from './mumbai'
-export * from './goerli'
-export * from './celo'
-export * from './arbitrum'
-export * from './avalanche'
-export * from './palm'
-export * from './base-goerli'
-export * from './base'
-export * from './linea'
-export * from './sepolia'
+export * from './localhost'
+export * from './mainnet'
+export * from './gnosis'
+export * from './polygon'
+export * from './bsc'
+export * from './optimism'
+export * from './mumbai'
+export * from './goerli'
+export * from './celo'
+export * from './arbitrum'
+export * from './avalanche'
+export * from './palm'
+export * from './base-goerli'
+export * from './base'
+export * from './linea'
+export * from './sepolia'
+export * from './pulsechain'
+export * from './pulsechainTestnetV4'

```

Comment:
We try to _not_ add all testnets because that means we have to maintain them.
Any reason why Goerli or Sepolia are not enough?

Reply:
All the development that is done on pulsechain is tried and tested using pulsechain testnet as it replicates the particular way in which the system state was forked over form Eth mainnet.

Reply:
We won't add it as it means more work for our team... if Pulsechain is not compatible with Ethereum test nets then we should not add support for it.